flink-ml-benchmark/src/test/java/org/apache/flink/ml/benchmark/BenchmarkTest.java

```diff
@@ -103,4 +166,32 @@ public void testRunBenchmark() throws Exception {
                 result.outputThroughput,
                 1e-5);
     }
+
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    private void dryRunBenchmark(Map<String, Map<String, ?>> params)
```

Comment:
It seems that this method duplicates a lot of existing logic implemented by `BenchmarkUtils#runBenchmark`. Would it be simpler to just add a `boolean isDryRun` parameter to that method?